### ISSUE TITLE
cc-env-var-editor-expert / cc-env-var-editor-json: hidden label values are messed up

### ISSUE THREAD

**Issue by florian-sanders-cc** [CleverCloud/clever-components#613]
# Context

As a temporary fix for missing labels, we have set visually hidden labels for `cc-env-var-editor-expert` and `cc-env-var-editor-json`.

# Issue

The values are passed as attributes but they are not plain strings.
They contain HTML and should be passed through JS properties instead of HTML attributes:
![image](https://user-images.githubusercontent.com/100240294/200590578-b85a9997-0c81-4c86-a088-43ded89d74c7.png)

# Solution

Use `.label` instead of `label` on the `cc-input-text` elements in the `cc-env-var-editor-expert` and `cc-env-var-editor-json`.
Or avoid using HTML in the label value. It's not really necessary anyway in this context.
